modules/nw-egress-ips-about.adoc

```diff
@@ -157,7 +157,9 @@ Egress IP addresses are implemented as additional IP addresses on the primary ne
 ====
 Egress IP addresses must not be configured in any Linux network configuration files, such as `ifcfg-eth0`.
 
-Allowing additional IP addresses on the primary network interface might require extra configuration when using some cloud or VM solutions.
+Egress IPs on cloud platforms is only supported on {product-title} 4.10.
```

Comment:
cloud platforms here means on  AWS,GCP,Azure, and not only 4.10, support  on 4.10 or later version.